### PR TITLE
Cleanup/Remove unnecessary whitespaces/new lines in scripts and documentation

### PR DESCRIPTION
**Pull request by galexrt**
Description of your changes:
* Removed a good amount whitespace "issues"/indentation in a lot of files.
* Ran beautify over `tests/` and `build/` scripts and resolved most shellcheck complaints.
* Made the markdown tables look better in the original file by changing the columns to full width.

Checklist:
- [x] Documentation has been updated, if necessary.
- [x] Pending release notes updated with breaking and/or notable changes, if necessary.
- [x] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.

### CODE REVIEW

**Comment by galexrt** [comment]
@jbw976 I don't know (not in our CI at least I think), but locally used probably for multi node tests (though due to the "nature" of docker containers without additional block storage, as for example my k8s-vagrant-multi-node would allow for, see #1540).
I pulled in the latest `master`.

Though I have to say I don't see a point in having the whole dind script in the repository, as it becomes outdated fast.

What about removing the dind script completely, and adding a script to simplify work with e.g. the new multi node environment in #1540? So we still have a way for multi node testing with ease locally.
(The script would contain functions identical as `minikube.sh` to simplify bootstrap of the environment)

What do you think?

**Comment by tonya11en** [comment]
@galexrt I think you should just go ahead and open an issue to remove the script and see who comments on it.

**Comment by galexrt** [comment]
@tonya11en @jbw976 I removed the removal of the dind-cluster script, will open an issue later on. PTAL

File: tests/README.md [hunk 1]
```diff
@@ -59,52 +58,51 @@ To use an alternate version, simply set this variable before running the relevan
...
 
-### Running Tests with parameters.
+#### 3. Run integration tests:
+Integration tests can be run using tests binary `_output/tests/${platform}/integration` that is generated during build time e.g.:
+```
+~/integration -test.v
+```
 
-#### To run all integration tests run 
+### Test parameters
```
Comment (jbw976) [inline tests/README.md]: This table is broken in the rendering on https://github.com/galexrt/rook/blob/583d156e5602a8d0390a15bc0925f1b93d03352a/tests/README.md#test-parameters.  You probably want to check all the other tables that were touched in this PR to ensure no others were broken.

See the original rendering that isn't broken here: https://github.com/rook/rook/blob/master/tests/README.md#test-parameters

**Comment by galexrt** [comment]
@jbw976 I fixed the table and resolved the merge conflicts. PTAL

**Pull request merged**